kb/additional-resources.md

```diff
@@ -1,14 +1,16 @@
 ---
-id: external-resources
-title: External Resources
-sidebar_label: External Resources
+slug: additional-resources
+title: Additional Resources
+tags:
+  - resources
+date: 2022-12-12T00:00:00Z
 ---
 
 There are many ways to introduce and teach Temporal based on what background you are coming from. Temporal doesn't have a monopoly on explaining Temporal.
 
 Here, we list great blogposts and videos by Temporal staff and users that may be helpful in increasing your understanding or convincing your team.
 
-If you have made a tutorial or explanation of Temporal, please [submit them here](https://github.com/temporalio/documentation/edit/master/docs/external-resources.md) so that others can find it!
+If you have made a tutorial or explanation of Temporal, please [submit them here](https://github.com/temporalio/documentation/edit/master/docs/kb/additional-resources.md) so that others can find it!
```

Comment:
```suggestion
If you have made a tutorial or explanation of Temporal, please [submit them here](https://github.com/temporalio/documentation/edit/master/kb/additional-resources.md) so that others can find it!
```